roles/network_plugin/weave/tasks/main.yml

```diff
@@ -18,5 +18,5 @@
 
 - name: Weave | Fix nodePort for Weave
   template:
-    src: 00-weave.conflist.j2
-    dest: /etc/cni/net.d/00-weave.conflist
+    src: 10-weave.conflist.j2
+    dest: /etc/cni/net.d/10-weave.conflist
```

Comment:
@mirwan this looks really silly for me...

* For new install with weave >= 2.5.0, weave's source code will handle the create of /etc/cni/net.d/10-weave.conflist internally, we don't need to additionally template it
* For upgrade to weave >= 2.5.0. the original file named as /etc/cni/net.d/00-weave.conflist, so will not conflict with the new version, no additional fix required (logically, they come with the same effect)

P.S. for manually deploy with older version < 2.5.0, our existing template CAN'T guarantee it could working well; at least during my maintenance and upgrade, I just keep sync templates with latest supported version :-(